### PR TITLE
Add license

### LINKED ISSUE #5: LICENSE is missing
Can we please have a license defined for this project?

cc @e-sumin @pavannd1

### PR DESCRIPTION
**Pull request by e-sumin**
Adding license.

closes #5

### CODE REVIEW

**Comment by denisvmedia** [comment]
@mlavi JFYI

**Comment by mlavi** [comment]
We need to add our copyright on Line 190, I'll comment there now.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (mlavi) [inline LICENSE]: Please change to: Copyright 2024 Kasten, Inc.

**Review by mlavi — CHANGES_REQUESTED** [review]
Please update line 190

File: LICENSE (hunk 1 above)
Comment (e-sumin) [reply LICENSE]: Done

File: LICENSE [hunk 2]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright Copyright 2024 Kasten, Inc.
```
Comment (denisvmedia) [inline LICENSE]: 2x Copyright.

Comment (mlavi) [reply LICENSE]: Almost there! :)

Comment (e-sumin) [reply LICENSE]: Thank you ! Fixed.

**Review by mlavi — APPROVED** [review]
LGTM

**Review by denisvmedia — APPROVED** [review]

**Pull request merged**